stmt/stmts/update.go

```diff
@@ -82,37 +85,27 @@ func (s *UpdateStmt) SetText(text string) {
 	s.Text = text
 }
 
-func getUpdateColumns(t table.Table, assignList []expressions.Assignment) ([]*column.Col, error) {
-	tcols := make([]*column.Col, len(assignList))
-	for i, asgn := range assignList {
+func getUpdateColumns(t table.Table, assignList []expressions.Assignment, multipleTable bool) ([]*column.Col, error) {
+	tcols := make([]*column.Col, 0, len(assignList))
+	tname := t.TableName()
+	for _, asgn := range assignList {
+		if multipleTable {
```

Comment:
For table `a(c1 int, c3 int), b(c2 int)`
then `update a, b set a.c1 = 1 and b.c2 = 2 and c3 = 3`,
can this be supported?